app.py

```diff
@@ -1,41 +1,78 @@
-from flask import Flask,request
+from flask import Flask, request
 from flask.json import jsonify
 
 app = Flask(__name__)
 
+
 class Availability:
     def __init__(self, start, end):
         self.start = start
         self.end = end
+
     def to_json(self):
         return {"start": self.start, "end": self.end}
 
-class Teacher:
-    def __init__(self, name, location, availabilities, price):
-        self.name = name
+
+class User:
```

Comment:
User is possibly the worst name you can imagine in a project. Especially that we will have two kind of users. 
It would make sense though if we would like to make the users interacting with a platform somehow symmetrical. The thing is you always come to the platform in one role and not the other. Let's discuss this further during the pair programming session